### DISCUSSION TITLE
How to Draw Background Axe

### DISCUSSION THREAD

**Discussion by a-reznic** [patrykandpatrick/vico#324] [Questions]
Something like this, it is posible?

<img width="517" alt="image" src="https://github.com/patrykandpatrick/vico/assets/3604074/c40a3722-c438-4104-8e15-fa6def02ffa8">

**Comment by af-hku** [comment]
- Create your own `Decoration` implementation
- Override `onDrawBehindChart` and draw whatever you want to be shown within it
- Use `decorations` parameter of `columnChart` or `lineChart` to add your decoration to your chart

(see https://patrykandpatrick.com/vico/wiki/extra-elements/)

**Reply by patrickmichalik** [reply]
Thanks for the input! It should, indeed, be possible to implement this as a `Decoration`. Another option is to create a custom `Axis` or `AxisRenderer` implementation. We plan on making `HorizontalAxis` more extensible.

This thread was previously an issue, and I mistakenly left a comment before converting the thread to a discussion. (Moving forward, we’d like to use discussions for everything but bug reports.) After the conversion, my comment turned into a separate answer, not a response to your answer, so I’m reposting my comment. Sorry for the extra notification.
